sdk/servicebus/azure-servicebus/README.md

```diff
@@ -100,13 +100,15 @@ The following sections provide several code snippets covering some of the most c
 * [Receive a message from a queue](#receive-from-a-queue)
 * [Defer a message on receipt](#defer-a-message)
 
+To perform management tasks such as creating and deleting queues/topics/subscriptions, please utilize the azure-mgmt-servicebus library, available [here][servicebus_management_repository].
+
 
 ### Send to a queue
 
 This example sends a message to a queue that is assumed to already exist, created via the Azure portal or az commands.
 
 ```Python
-from azure.servicebus import ServiceBusClient
+from azure.servicebus import ServiceBusClient, Message
```

Comment:
`as queue_sender` is missed in this send example:
with client.get_queue_sender(queue_name) **as queue_sender:**
```
    with client.get_queue_sender(queue_name):
        message = Message("Single message")
        queue_sender.send(message)
```

Reply:
++ good eyes.  I did what I should have done at the getgo and validated this run in an individual file (added another envvar for queuename as such.)